### ISSUE TITLE
Build simple neural network using JAX

### ISSUE THREAD

**Issue by RUBIKX107** [RUBIKX107/Jax.exe#2]
Goal: Create a small feedforward network.

Tasks:
- Initialize weights
- Forward pass
- Loss function
- Backprop using JAX gradients

**Comment by RUBIKX107** [comment]
Created an NN using Jax with That i learned that the loss getting lower from step 0-step 49 , what we can understand that’s the inexperience gets better and with that the loss getting down . 

The steps that i made were: 
Step 1 - created the main.py 
Step 2 - i install the needed rescoures **pip install Jax Jaxlib**
Step 3 - used the code as written in the branch 
Step 4 - run with **python main.py** 
Step 5 - checked the results and the differences . 
Step 6 - for saving have to use **git checkout** 

With that we can see that the issue is worked and solved
